src/main/java/com/austinv11/servicer/ServicerProcessor.java

```diff
@@ -115,4 +113,22 @@ public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment
     public SourceVersion getSupportedSourceVersion() {
         return super.getSupportedSourceVersion();
     }
+
+    private static class Services {
+        private final Set<String> impls = new HashSet<>();
+        private final Set<Element> elements = new HashSet<>();
```

Comment:
Are you sure it's safe to hold Elements across processing rounds? (I am unsure personally).

Reply:
I am not

Reply:
I'll check though.

Reply:
It looks like dagger avoids reusing, so I'm going to use something else.